content/docs/5.1/concepts/explore-tikv-features/distributed-transaction.md

```diff
@@ -123,13 +124,13 @@ python3 test_optimistic.py
 Exception: KeyError WriteConflict
 ```
 
-As we can see, with optimistic transactions conflicting changes are detected when the transaction commits.
+From the above example, you can find that with optimistic transactions, conflicting changes are detected when the transaction commits.
 
-## Step 3: Try pessimistic transaction model
+## Try pessimistic transaction model
 
 In the optimistic transaction model, transactions might fail to be committed because of write–write conflict in heavy contention scenarios. In the case that concurrent transactions frequently modify the same rows (a conflict), pessimistic transactions may perform better than optimistic transactions.
```

Comment:
```suggestion
In the optimistic transaction model, transactions might fail to be committed because of write–write conflict in heavy contention scenarios. In the case that concurrent transactions frequently modify the same rows (a conflict), pessimistic transactions might perform better than optimistic transactions.
```